target-platform/p2-repo-common/pom.xml

```diff
@@ -82,6 +82,11 @@
 									<version>${org.eclipse.kura.sun.misc.version}</version>
 								</artifactItem>
 								<artifactItem>
+								    <groupId>org.json</groupId>
```

Comment:
As already mentioned earlier, for a new component please use some JSON library which is OSGi ready on maven central and does not have legal issues.